### PR TITLE
Game knows when over

### PR DESCRIPTION
**Pull request by ksabbak**

### CODE REVIEW

File: src/clojure_tictactoe/game/game_rules.clj [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  [board]
+  (cond 
+    (= (board 0) (board 1) (board 2)) (board 0)
+    (= (board 3) (board 4) (board 5)) (board 3)
+    (= (board 6) (board 7) (board 8)) (board 6)
+    (= (board 0) (board 3) (board 6)) (board 0)
+    (= (board 1) (board 4) (board 7)) (board 1)
+    (= (board 2) (board 5) (board 8)) (board 2)
+    (= (board 0) (board 4) (board 8)) (board 0)
+    (= (board 2) (board 4) (board 6)) (board 2)))
```
Comment (hnlee) [inline src/clojure_tictactoe/game/game_rules.clj:15]: This works for now, but I'd like to see you refactor this later. It's pretty verbose and I bet you can get it to be simpler and clearer.

**Review by hnlee — COMMENTED** [review]
Unit testing is much improved!

File: src/clojure_tictactoe/game/game_rules.clj (hunk 1 above)
Comment (ksabbak) [reply src/clojure_tictactoe/game/game_rules.clj:15]: I agree.

Comment (BWoodfork) [reply src/clojure_tictactoe/game/game_rules.clj:15]: +1. What's the plan for implementing the refactor for this?

**Pull request merged**